specification/Metadata/Semantics/README.md

```diff
@@ -0,0 +1,190 @@
+
+## Contents
+
+- [Cesium Metadata Semantic Reference](#cesium-metadata-semantic-reference)
+    - [Overview](#overview)
+  - [Contents](#contents)
+  - [General](#general)
+    - [Overview](#overview-1)
+    - [`ID`](#id)
+    - [`NAME`](#name)
+  - [3D Tiles](#3d-tiles)
+    - [Overview](#overview-2)
+    - [Tile](#tile)
+      - [Overview](#overview-3)
+      - [`TILE_BOUNDING_BOX`](#tile_bounding_box)
+      - [`TILE_BOUNDING_REGION`](#tile_bounding_region)
+      - [`TILE_BOUNDING_SPHERE`](#tile_bounding_sphere)
+      - [`TILE_MINIMUM_HEIGHT`](#tile_minimum_height)
+      - [`TILE_MAXIMUM_HEIGHT`](#tile_maximum_height)
+      - [`TILE_HORIZON_OCCLUSION_POINT`](#tile_horizon_occlusion_point)
+      - [`TILE_GEOMETRIC_ERROR`](#tile_geometric_error)
+    - [Content](#content)
+      - [Overview](#overview-4)
+      - [`CONTENT_BOUNDING_BOX`](#content_bounding_box)
+      - [`CONTENT_BOUNDING_REGION`](#content_bounding_region)
+      - [`CONTENT_BOUNDING_SPHERE`](#content_bounding_sphere)
+      - [`CONTENT_MINIMUM_HEIGHT`](#content_minimum_height)
+      - [`CONTENT_MAXIMUM_HEIGHT`](#content_maximum_height)
+      - [`CONTENT_HORIZON_OCCLUSION_POINT`](#content_horizon_occlusion_point)
+  - [Revision History](#revision-history)
+
+## General
+
+### Overview
+
+Throughout this section, the term "entity" refers to any conceptual object with which a property value (as defined in the [Cesium 3D Metadata Specification](../)) may be associated. Examples of entities include tilesets, tiles, and tile contents in 3D Tiles, or groups of vertices and texels in glTF 2.0 assets. Additional types of entities may be defined by other specifications or applications.
+
+### `ID`
+
+The unique identifier for the entity.
+
+* Type: `STRING`
+
+### `NAME`
+
+The name of the entity. Names should be human-readable, and do not have to be unique.
+
+* Type: `STRING`
+
+## 3D Tiles
+
+### Overview
+
+Semantics for 3D Tiles are assigned in relationship to a tile or tile content, as defined by the 3D Tiles 1.0 specification. When associated with other types of entities, these semantics may have invalid or undefined meanings.
+
+Units for all linear distances are meters, and all angles are radians.
+
+### Tile
+
+#### Overview
+
+`TILE_*` semantics provide meaning for properties associated with a particular tile, and should take precedence over equivalent metadata on parent objects, as well as over values derived from subdivision schemes like [3DTILES_implicit_tiling](../../../extensions/3DTILES_implicit_tiling).
+
+In particular, `TILE_BOUNDING_BOX`, `TILE_BOUNDING_REGION`, and `TILE_BOUNDING_SPHERE` semantics each define a tighter bounding volume for a tile than is implicitly calculated from [3DTILES_implicit_tiling](../../../extensions/3DTILES_implicit_tiling). If more than one of these semantics are available for a tile, clients may select the most appropriate option based on use case and performance requirements.
```

Comment:
> tighter bounding volume

Not sure if we should say tighter since there are some cases where it may be larger, like using `TILE_BOUNDING_SPHERE` when the implicit root tile is a region. Though it's still the most common use case so it should be mentioned, just in a different way.

Reply:
> a more specific bounding volume...

This wording sounds good to me

Reply:
```suggestion
In particular, `TILE_BOUNDING_BOX`, `TILE_BOUNDING_REGION`, and `TILE_BOUNDING_SPHERE` semantics each define a more specific bounding volume for a tile than is implicitly calculated from [3DTILES_implicit_tiling](../../../extensions/3DTILES_implicit_tiling). If more than one of these semantics are available for a tile, clients may select the most appropriate option based on use case and performance requirements.
```